test/integration/golden_response/staging/get_stat_all/result.json

```diff
@@ -121,15 +122,16 @@
                 "2007": 301231207,
                 "2008": 304093966,
                 "2009": 306771529,
-                "2010": 309326085,
-                "2011": 311580009,
-                "2012": 313874218,
-                "2013": 316057727,
-                "2014": 318386421,
-                "2015": 320742673,
-                "2016": 323071342,
-                "2017": 325147121,
-                "2018": 327167434
+                "2010": 309321666,
+                "2011": 311556874,
+                "2012": 313830990,
+                "2013": 315993715,
+                "2014": 318301008,
+                "2015": 320635163,
+                "2016": 322941311,
+                "2017": 324985539,
+                "2018": 326687501,
+                "2019": 328239523
```

Comment:
Wow this is surprising... we were missing 10 years before?

Reply:
The delta is showing up poorly, past 10 years just changed, but existed?

Reply:
Oh I see. Thanks!